content/en/blog/_posts/2022-08-16-PSP-historical-context/index.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: blog
+title: "PodSecurityPolicy: The Historical Context"
+date: 2022-08-16
+slug: podsecuritypolicy-the-historical-context
+---
+
+**Author:** Mahé Tardy (Quarkslab)
+
+The PodSecurityPolicy (PSP) admission controller will be removed as of
+Kubernetes v1.25. Its deprecation was announced and detailed in the blog post
+[PodSecurityPolicy Deprecation: Past, Present, and Future](/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future/),
+published for the Kubernetes v1.21 release.
+
+This article aims at providing historical context on the birth and evolution of
+PSP, to explain why the feature never made it to stable and had to be removed
+in favor of a new in-tree admission controller. Let's recap what
+PodSecurityPolicy is, why it was necessary, the historical context of its
+addition to Kubernetes, and finally, why it was removed and replaced by Pod
+Security admission control.
+
+PodSecurityPolicy, like other specialized admission control plugins, provided
+fine-grained permissions on specific fields concerning the pod security
+settings as a built-in policy API. It acknowledged the fact that cluster
+administrators and cluster users are, most of the time, not the same people, and
+that creating workloads in the form of a Pod or any resource that will
+create a Pod should not equal being "root on the cluster". PodSecurityPolicy
```

Comment:
```suggestion
that creating workloads in the form of a Pod or any resource that
creates a Pod should not require being "root on the cluster". PodSecurityPolicy
```

Reply:
This suggestion will change the idea that I want to express: the problem I want to explain here is that "if you are able to create a workload in a cluster without admission control, you can be root on any node of the cluster, i.e. root on the cluster" and not "you should not require being root on the cluster to be able to create a workload".

Do you think my suggestion is unclear or confusing in the first place? We could explain it more clearly!